buddyboss/buddyboss-platform
# Auto generate preview link after insert Emoji and enter new link

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1331 

### How to test the changes in this Pull Request:

1. Go to timeline or group feed form Desktop
2. Then try to post something and insert Emoji then Enter for the new line
3. See the auto-generate a preview link


### Proof Screenshots or Video
http://prntscr.com/txof6w


### Other information:
